Configurations/90-team.conf

```diff
@@ -100,6 +116,8 @@ my %targets = (
     },
     "darwin64-debug-test-64-clang" => {
         inherit_from     => [ 'BASE_unix', "x86_64_asm" ],
+        policy           => 'primary', # speculative
```

Comment:
These do seem pretty speculative.
While I don't think it's wrong to leave them as 'primary', I would also see a case for marking them 'deprecated'.